dotnet/dotnet-cloud-run-hello-world/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine AS base
+WORKDIR /src
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS build
+WORKDIR /src
+COPY . .
+RUN dotnet restore dotnet-cloud-run-hello-world.csproj
+RUN dotnet build "./dotnet-cloud-run-hello-world.csproj" -c Debug -o /out
+
+FROM build AS publish
+RUN dotnet publish dotnet-cloud-run-hello-world.csproj -c Debug -o /out
+
+# Building final image used in running container
+FROM base AS final
+WORKDIR /src
+COPY --from=publish /out .
+ENV ASPNETCORE_URLS=http://*:8080
```

Comment:
Hard-coding 8080 is not compliant with the runtime contract. We want the service to bind on $PORT environment variable at runtime. If we must specify 8080 here, we should also deploy with `--port 8080`, but that's really intended to support services developers cannot modify.

Reply:
Very good point, thanks! It should have the port specified via $PORT.